src/controllers/usersController.js

```diff
@@ -0,0 +1,39 @@
+import UsersService from '../services/usersService';
+import Response from '../utils/response';
+
+/**
+ * Class for interacting with User profile information
+ */
+export default class UsersController {
+/**
+ * @param {object} user - user auth token payload
+ * @param {object} res - response object
+ * @return {object} user - return object containing status and data
+ */
+  static async getProfile({ user }, res) {
+    try {
+      const { id } = user;
+      const userData = await UsersService.getProfile(id);
+
+      Response.success(res, userData);
```

Comment:
Destruction success and badRequest from Response. That is, `const { success, badRequest} = Response`
So you can have `sucess(res, userData)`.  You also apply this in the other places where i applies. I This think this will make your code cleaner